JavaScript/Topics/11.Scheduling/README.md

```diff
@@ -0,0 +1,78 @@
+# Scheduling
+
+There are primarily two ways of scheduling a task in JavaScript - ```setTimeout()``` and ```setInterval()```.
+<br>
+
+Both ```setTimeout()``` and ```setInterval()``` are built-in methods of the global object on the Document Object Model to schedule tasks at a set time.```setTimeout()``` calls a passed-in function once after a specified delay, while ```setInterval()``` invokes one continuously at a designated time.
+<br>
+
+- ## **setTimeout()**
+  The ```setTimeout``` function is a native JavaScript function. It sets a timer (a countdown set in milliseconds) for an execution of a callback function, calling the function upon completion of the timer.
+
+  The ```setTimeout()``` method accepts two arguments: a callback function and a delay in milliseconds, and calls the function once.
```

Comment:
Specify the syntax of setTimeout and setInterval,

Like this -> setTimeout(func(), timeInMilliseconds)

Reply:
@creatoon done as you said

Reply:
then resolve this, so that when others review will not have to face these comments